administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -622,4 +635,16 @@ public function getItems()
 
         return $items;
     }
+
+    /**
+     * Method to get the value of featured selector.
+     *
+     * @return  string  Returns the value of featured selector.
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public function isFeatured()
+    {
+        return $this->getUserStateFromRequest($this->context . '.featured', 'featured', 'int');
```

Comment:
filter.featured has been removed, preventing quickicons, for instance, to properly count the number of featured articles.
```suggestion
        $featured = (string) $this->getState('filter.featured', '');
        if ($featured !== '') {
            return (int) $featured;
        }
        return $this->getUserStateFromRequest($this->context . '.featured', 'featured', 'int');
```